pnp-helper/src/main/java/samples/com/microsoft/azure/sdk/iot/helper/PnpHelper.java

```diff
@@ -0,0 +1,73 @@
+package samples.com.microsoft.azure.sdk.iot.helper;
+
+import com.google.gson.JsonObject;
+import com.microsoft.azure.sdk.iot.service.devicetwin.*;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+public class PnpHelper {
+    public static Set<Pair> CreatePropertyPatch(String propertyName, String propertyValue)
+    {
+        // The update patch for a property on the root level or a device without component should be in the format:
+        // "desired": {
+        //    "propertyName": {
+        //      "value": "hello"
+        //  }
+        Set<Pair> desiredProperties = new HashSet<Pair>();
+        desiredProperties.add(new Pair(propertyName, propertyValue));
+        return desiredProperties;
```

Comment:
nit: this can also be done in a single statement:
```java
return Collections.singleton(new Pair(properyName, propertyValue));
```

Reply:
Cool! Good to know :)